modules/model/shared/src/main/scala/observe/model/SequenceState.scala

```diff
@@ -17,24 +17,25 @@ import monocle.macros.GenPrism
 enum SequenceState(val name: String) derives Eq, Encoder, Decoder:
   case Idle                extends SequenceState("Idle")
   case Running(
-    userStop:            Boolean,
-    internalStop:        Boolean,
-    isWaitingUserPrompt: Boolean,
-    isStarting:          Boolean
+    userStop:          Boolean,
```

Comment:
It maybe good to document these fields. Especially as they are all booleans

Reply:
I'll switch to newtypes in a future PR.